Assignment 3 Instructions.Rmd

```diff
@@ -160,10 +167,85 @@ Once you have done this, save both of your plots to the Assignment 5 file. Creat
 
 Now, try to do the same for the data [collected in class](https://tccolumbia.qualtrics.com/SE/?SID=SV_6RRqlSAM6lZWYQt). Create two groups of clusters, the answers to the questions and regions where people grew up. 
 
+#First Part of Part II - create a cluster of the answers to the questions.
+#1. convert data to data frame (K8)
+#2. omit names and select the rest of the columns, 3 to 12
+#3. convert column Cat and Gif to 1s and 2s: ifelse(df$column=="word",1,2)
+#3. standarize the range w/scale.
+
+```{r}
+K8 <- data.frame(HUDK4050_2017_Cluster_Survey)
+K9 <- dplyr::select(K8, 3:12)
+K9$cat <- ifelse(K9$cat=="Yes",1,2)
+K9$gif.jiff <- ifelse(K9$gif.jiff=="g-iff",1,2)
+Komit <- na.omit(K9)
+K9 <- scale(K9) 
+
+#4. create cluster - let's choose 2 clusters (for now :))
+#5. Attach these clusters to K9
+fit2<-kmeans(K9, 5)
+K10 <- data.frame(K9, fit2$cluster)
+```
+
+```{r}
+
+library(cluster)
+clusplot(K9, fit2$cluster, color=TRUE, labels=1, lines=0, main="Dr. Lang's Favorite Class Data Results")
+
+```
+```{r}
+#Second Part of Part II - Regions where people grew up
+#1. create K.regions data frame to isolate the regions
+#2. select the column for the city (for now, add state once it works :))
+#3. use ggmap & maps
+
+K.regions <- dplyr::select(K8, 15)
+library("ggmap", lib.loc="C:/Program Files/R/R-3.4.1/library")
+library("maps", lib.loc="C:/Program Files/R/R-3.4.1/library")
+
+#change United States to USA (this took FOREVER to figure out! Tried replace, rename)
```

Comment:
Glad you worked it out though! 👏👏👏 Although you do not need to re-code each variable if you are using the original code. Just use the original variable name, eg. `ifelse(K.regions$country.grew == "United States", "USA", K.regions$country.grew)`